### PR TITLE
what are these pussy ass bombs LMAO 😂😂😂🍆💦😍

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
experimental: where the boom @
/:cl:

### CODE REVIEW

**Comment by calzilla1** [comment]
Put me in the screen shot

**Comment by Buggy123** [comment]
Sound good to me merge asap.

**Review by Tlaltecuhtli — APPROVED** [review]

**Comment by ishitbyabullet** [comment]
Crank that heavy range to 1000.

**Comment by awesine** [comment]
put me in the screenshot

**Comment by awesine** [comment]
also remove the bombcap

**Review by ghost — APPROVED** [review]
looks good, no errors, and a beneficial and gameplay-improving change.

**Review by DaedalusGame — APPROVED** [review]
Balanced as bombs can't be put into backpacks anymore, this will make it so we can carbomb security again by blowing the bomb in toxins.

File: code/_globalvars/configuration.dm [hunk 1]
```diff
@@ -23,14 +23,14 @@ GLOBAL_VAR_INIT(Debug2, FALSE)
...
 //This was a define, but I changed it to a variable so it can be changed in-game.(kept the all-caps definition because... code...) -Errorage
 //Protecting these because the proper way to edit them is with the config/secrets
-GLOBAL_VAR_INIT(MAX_EX_DEVESTATION_RANGE, 3)
+GLOBAL_VAR_INIT(MAX_EX_DEVESTATION_RANGE, 15)
 GLOBAL_PROTECT(MAX_EX_DEVESTATION_RANGE)
-GLOBAL_VAR_INIT(MAX_EX_HEAVY_RANGE, 7)
+GLOBAL_VAR_INIT(MAX_EX_HEAVY_RANGE, 35)
 GLOBAL_PROTECT(MAX_EX_HEAVY_RANGE)
-GLOBAL_VAR_INIT(MAX_EX_LIGHT_RANGE, 14)
+GLOBAL_VAR_INIT(MAX_EX_LIGHT_RANGE, 70)
```
Comment (kevinz000) [inline code/_globalvars/configuration.dm]: What the fuck dude, this is literally, LITERALLY nearly half the station.

Comment (yorii) [reply code/_globalvars/configuration.dm]: kek

Comment (ACCount12) [reply code/_globalvars/configuration.dm]: Awesome, right?

**Comment by Buggy123** [comment]
T W E N T Y F O U R H O U R R U L E

**Comment by awesine** [comment]
T W E N T Y F O U R H O U R R U L E

**Comment by DaedalusGame** [comment]
it's been 3 days also you're banned go away

**Comment by awesine** [comment]
shut up nerd

**Pull request closed without merging**